sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -743,6 +743,20 @@ def pcollection_count_monitoring_infos(self, tag_to_pcollection_id):
     return infos
 
 
+class SdfTruncateSizedRestrictions(DoOperation):
+  def __init__(self, *args, **kwargs):
+    super(SdfTruncateSizedRestrictions, self).__init__(*args, **kwargs)
+
+  def current_element_progress(self):
+    # type: () -> Optional[iobase.RestrictionProgress]
+    return self.consumers[0][0].current_element_progress()
+
+  def try_split(
+      self, fraction_of_remainder
+  ):  # type: (...) -> Optional[Tuple[Iterable[SdfSplitResultsPrimary], Iterable[SdfSplitResultsResidual]]]
+    return self.consumers[0][0].try_split(fraction_of_remainder)
```

Comment:
```suggestion
    return self.receivers[0].try_split(fraction_of_remainder)
```

We should be relying on the ConsumerSet to properly forward the split/progress call. See: 
https://github.com/apache/beam/blob/af2d6b0379d64b522ecb769d88e9e7e7b8900208/sdks/python/apache_beam/runners/worker/operations.py#L150
https://github.com/apache/beam/blob/af2d6b0379d64b522ecb769d88e9e7e7b8900208/sdks/python/apache_beam/runners/worker/operations.py#L198